algorithms.js

```diff
@@ -211,3 +211,31 @@ function* shellSort(values) {
     }
   }
 }
+
+function* radixSortLSD(array) {
+  let values = array.slice(0, array.length);
+  let counter = Array.from({ length: 10 }).fill(0, 0);
```

Comment:
hmm how does this work? Can it be replaced by: 

```suggestion
  let counter = new Array(10).fill(0);
```